components/event-footer/EventFooter.vue

```diff
@@ -31,7 +31,13 @@
 
 <style>
   .event-footer {
-    background: linear-gradient(rgba(0,0,0,0.7), rgba(0,0,0,0.9));
```

Comment:
Is the gradient included in the image now?
It increases the image size and in the design the gradient is slightly different per viewport, that's why it's done in CSS.

Reply:
Yes, but it didn't play nice with the background: the gradient is almost, but not really, the same color as the background of the page.